Formula/grafana.rb

```diff
@@ -0,0 +1,154 @@
+require "language/node"
+
+class Grafana < Formula
+  desc "Gorgeous metric visualizations and dashboards for timeseries databases."
+  homepage "http://grafana.org"
+  url "https://github.com/grafana/grafana/archive/v3.0.4.tar.gz"
+  sha256 "f26a374326e64a8f83c57fdf916ba1c8524dd55002dfe628b4854d05fed3715a"
+
+  head "https://github.com/grafana/grafana.git"
+
+  depends_on "go" => :build
+  depends_on "node" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    grafana_path = buildpath/"src/github.com/grafana/grafana"
+    grafana_path.install Dir["*"]
+    grafana_path.install ".jscs.json", ".jsfmtrc", ".jshintrc", ".bowerrc"
+
+    cd grafana_path do
+      # The sass-lint npm package dependencey in package.json specifies any
+      # version greater than 1.6.0 for grafana, but on OS X versions 1.8.0+
+      # break. This replaces the specification for any version >= 1.6.0 and
+      # sets it to 1.7.0 (which works).
+      inreplace "package.json", '"sass-lint": "^1.6.0"', '"sass-lint": "1.7.0"'
+      system "go", "run", "build.go", "setup"
+      system "go", "run", "build.go", "build"
+      system "npm", "install", *Language::Node.local_npm_install_args
+      system "npm", "install", "grunt-cli", *Language::Node.local_npm_install_args
+      system "node_modules/grunt-cli/bin/grunt", "build"
+
+      bin.install "bin/grafana-cli"
+      bin.install "bin/grafana-server"
+      (bin/"grafana").write(env_script)
+      chmod 0755, bin/"grafana"
+      (etc/"grafana").mkpath
+      cp("conf/sample.ini", "conf/grafana.ini.example")
+      unless (etc/"grafana/grafana.ini").exist?
+        etc.install "conf/sample.ini" => "grafana/grafana.ini"
+      end
+      etc.install "conf/grafana.ini.example" => "grafana/grafana.ini.example"
+      pkgshare.install Dir["conf", "vendor"]
+      pkgshare.install "public_gen" => "public"
+    end
+  end
+
+  def post_install
+    (var/"log/grafana").mkpath
+    (var/"lib/grafana/plugins").mkpath
+  end
+
+  def env_script
```

Comment:
Can you remove the `env_script` in favour of letting people call `grafana-server` directly and see if you can get upstream to accept and bundle this script with the application?